samples/sample_vpp/src/sample_vpp_utils.cpp

```diff
@@ -414,6 +426,107 @@ mfxStatus InitParamsVPP(mfxVideoParam* pParams, sInputParams* pInParams, mfxU32
 
 /* ******************************************************************* */
 
+#if (defined(_WIN64) || defined(_WIN32)) && (MFX_VERSION >= MFX_VERSION_NEXT)
+mfxU32 GetPreferredAdapterNum(const mfxAdaptersInfo & adapters, const sInputParams & params)
```

Comment:
emmmm.....

Reply:
these are different input parameters, but they have same name. We shouldn't bring this to utility, since we don't want to put giant structs of input parameters description to common code

Reply:
I got impression that you use just 1, maybe 2 values from input parameters. If so, you can change function prototype and pass only what you need making it possible to move function to utilities library.

Reply:
Not sure if it makes sense. In this case it would be something like:
```
common_header.h
...
mfxU32 GetPreferredAdapterNum(const mfxAdaptersInfo & adapters, const mfxVideoParam& params)
```

But in each sample:
```
sample.cpp
...
void FillParameters(const sInputParams & input_params, mfxVideoParam& output_params)
```

Doesn't look much better